rules/ensure-default-service-accounts-has-only-default-roles/raw.rego

```diff
@@ -0,0 +1,33 @@
+package armo_builtins
+
+
+# deny if a default ServiceAccount has rules bound to it that are not defaults. 
+deny[msga] {
+
+    wl := input[_]
+	spec_template_spec_patterns := {"RoleBinding", "ClusterRoleBinding"}
+	spec_template_spec_patterns[wl.kind]
+
+    # filter service accounts
+    wl.subjects[_].kind == "ServiceAccount"
+
+    # filter out defaults
+    wl.subjects[_].name != "default"
```

Comment:
```suggestion
    wl.subjects[i].kind == "ServiceAccount"

    # filter out defaults
    wl.subjects[i].name != "default"
```
you want to make sure this is the same subject

Reply:
Fixed.